test/documentation/api-stability.js

```diff
@@ -13,12 +13,28 @@ $(document).ready(function()
         $(this).find('div.memproto').html(new_html);
     }
 
-    var add_labels = function(api_type, entry_type)
+    var add_labels = function(api_type, api_color, entry_obj, entry_type)
     {
         if (entry_type == "member")
+        {
             $(this).find('span.mlabels').prepend('<span class="mlabel ' + api_type + '">' + api_type + '</span>');
+
+            // get header above the item, append the api_type tag to the header and change the bg color
+            prev = entry_obj;
+            do prev = prev.previousSibling; while(prev && prev.nodeType !== 1);
+
+            prev.innerContent = prev.innerContent + '  ' + '[' + api_type + '] ';
+
+            if (api_type != "stable-api") // for now, leave stable api background in blue.
+            {
+                prev.style.backgroundImage = "none";
+                prev.style.backgroundColor = api_color;
+            }
```

Comment:
I would prefer if we added a css class here and make the styling in the css file. That would have the result that you don't need to pass in the color.

Reply:
more work for something so tiny? :(